tools/Dashboard/DevHome.Dashboard/ComSafeWidgetObjects/ComSafeWidget.cs

```diff
@@ -359,4 +371,20 @@ public static async Task<string> GetIdFromUnsafeWidgetAsync(Widget widget)
             return string.Empty;
         });
     }
+
+    public Widget GetUnsafeWidgetObject()
+    {
+        return _oopWidget;
+    }
+
+    // CoAllowSetForegroundWindow must be called on a raw COM interface, not a .NET CCW, in order to work correctly, since
+    // the underlying functionality is implemented by COM runtime and the object itself. CoAllowSetForegroundWindow wrapper
+    // below takes a WinRT object and extracts the raw COM interface pointer from it before calling native CoAllowSetForegroundWindow.
+    [DllImport("ole32.dll", ExactSpelling = true, PreserveSig = false, SetLastError = true)]

```

Comment:
nit: nativemethods.txt

Reply:
+1.  But checking this in now for Canary this week.